### PR TITLE
secure ui when activating billing of transactions

### PR DESCRIPTION
**Pull request by cabch**

### CODE REVIEW

File: src/app/pages/settings-integration/billing/stripe/settings-stripe.component.html [hunk 1]
```diff
@@ -5,9 +5,11 @@
     </div>
     <h3 class="card-title">{{'settings.billing.stripe.title' | translate}}</h3>
   </div>
-
   <div class="card-body">
     <div class="row mt-1">
+      <div *ngIf="transactionBillingActivated" class="col-md-6 deactivated-setting-message">
```
Comment (ClaudeROSSI) [inline src/app/pages/settings-integration/billing/stripe/settings-stripe.component.html:10]: This has a side effect on the position of the first the url. Can you change it?

image

File: src/assets/i18n/de.json [hunk 2]
```diff
@@ -1493,6 +1493,7 @@
     },
     "billing": {
       "description": "Das Abrechnungsinterface ermöglicht Abrechnungsfunktionen für den Ladestationenbetreiber.",
+      "deactivated_setting_message": "Billing of transactions is activated, you cannot change the billing information anymore",
```
Comment (ClaudeROSSI) [inline src/assets/i18n/de.json]: the term transaction is confusing (and only used internally). The UI always uses the term "sessions" (or charging sessions).

I'll make a proposal with different terms.

**Review by ClaudeROSSI — CHANGES_REQUESTED** [review]
small side effect on the position of the url (to stripe).
Can you change it? 

I pushed some changes to replace the term "transactions" by the term "charging sessions"

File: src/app/pages/settings-integration/billing/stripe/settings-stripe.component.html (hunk 1 above)
Comment (cabch) [reply src/app/pages/settings-integration/billing/stripe/settings-stripe.component.html]: done ✅

File: src/assets/i18n/de.json (hunk 2 above)
Comment (cabch) [reply src/assets/i18n/de.json]: ok 👍🏻

**Review by ClaudeROSSI — APPROVED** [review]

**Pull request merged**